openapi/components/schemas/Applications/ApplicationInstance.yaml

```
@@ -12,16 +12,21 @@ properties:
       - enabled
       - disabling
       - disabled
-  settings:
+  isConfigured:
     description: |-
-      Settings that are available for the user to complete when they install the application.
-      Use this object to describe fields that are rendered during installation, in the settings form.
+      Determines if the application instance configuration complete.
+      Configuration is done by [PutApplicationInstanceConfiguration](https://all-rebilly.redoc.ly/tag/Application-users/operation/PutApplicationInstanceConfiguration).
```

suggestion: suggestion for readability 
```suggestion
      Configuration is completed using the [PutApplicationInstanceConfiguration](https://all-rebilly.redoc.ly/tag/Application-users/operation/PutApplicationInstanceConfiguration) operation.
```